Jantimizer/QueryPlanParser/Models/AnalysisResult.cs

```diff
@@ -15,31 +15,23 @@ public class AnalysisResult
         public ulong ActualCardinality { get; }
         public TimeSpan ActualTime { get; }
         [Ignore]
-        public List<AnalysisResult> SubQueries { get; }
+        public AnalysisResultQueryTree QueryTree { get; }
 
-        public AnalysisResult(string name, decimal estimatedCost, ulong estimatedCardinality, ulong actualCardinality, TimeSpan actualTime)
+        public AnalysisResult(AnalysisResultQueryTree subTree)
```

Comment:
Why does the only constructor for this require another model? Why cant it just be the direct property values?